src/layouts/CustomerInvoiceDetails.tsx

```diff
@@ -193,6 +209,35 @@ const mapStatus = (
   return { label: 'n/a', type: StatusType.default }
 }
 
+const getErrorMessageFromErrorDetails = (
+  errors: AllInvoiceDetailsForCustomerInvoiceDetailsFragment['errorDetails'],
+): string | undefined => {
+  if (!errors || errors.length === 0) {
+    return undefined
+  }
+
+  const [{ errorCode, errorDetails }] = errors
+
+  if (errorCode === ErrorCodesEnum.TaxError) {
+    if (errorDetails === LagoApiError.CurrencyCodeNotSupported) {
+      return 'Currency defined is not supported for taxes calculation. Please contact the Lago team to resolve this issue.'
+    }
+
+    if (
+      errorDetails === LagoApiError.CustomerAddressCouldNotResolve ||
+      errorDetails === LagoApiError.CustomerAddressCountryNotSupported
+    ) {
+      return 'Customer address information has issues preventing calculating taxes. Please update the information to generate the invoice.'
+    }
+
+    if (errorDetails === LagoApiError.ProductExternalIdUnknown) {
+      return 'Anrok connection items mapping has issues preventing calculating taxes. Please update the mapping to generate the invoice.'
+    }
+
+    return 'An issue with your tax provider connection occurred. Please contact the Lago team to solve this issue.'
+  }
+}
+
```

Comment:
TODO: Replace with https://github.com/getlago/lago-front/blob/8e484a5488ee9e45c554672477f4ea0012f7784a/src/core/constants/form.ts#L73-L78 when it's available